src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/security/authentication/GithubAuthenticator.kt

```diff
@@ -54,7 +54,7 @@ class GithubAuthenticator(private val userRepository: UserRepository,
         val githubOrg = appConfig["auth.github_org"]
         val teamName = appConfig["auth.github_team"]
 
-        if (!githubOrg.isEmpty() && !userBelongToOrg(githubOrg, user))
+        if (!githubOrg.isEmpty() && !currentUserBelongsToOrg(githubOrg))
         {
             throw CredentialsException("User is not a member of GitHub org $githubOrg or token does not include read:org scope")
```

Comment:
I guess this message could now be amended to just "User is not a member of GitHub org $githubOrg" seeing as the second disjunct would result in a CredentialsException being thrown already